components/footer.tsx

```diff
@@ -1,9 +1,15 @@
 import { Nav } from "react-bootstrap";
 import styles from "../styles/etc.module.css";
 
-const Footer = () => {
-    return(
-        <Nav className={`sticky-bottom justify-content-center mt-3 ${styles.footerNav}`}>
+interface FooterProps {
+    style?: string
```

Comment:
instead of passing a string which can be an arbitrary class name, you should use an enum to constrain valid inputs.

for example:

```
enum FooterPosition {
  Sticky,
  Fixed,
}

interface FooterProps {
  position?: FooterPosition
}
```

then the component would look like:
```
const Footer = ({position = FooterPosition.Sticky}: FooterProps) => {
  const positionClasses = {
    [FooterPosition.Sticky]: 'sticky-bottom',
    [FooterPosition.Fixed]: 'fixed-bottom',
  };

  return (
    <Nav className={`${positionClasses[position]} ...`}>
       ...
  );
}
```

then you could say:

```
<Footer position={FooterPosition.Fixed} />
```